include/zephyr/drivers/spi.h

```diff
@@ -149,11 +227,17 @@ extern "C" {
 
 /** @} */
 
+/**
+ * @name SPI GPIO Chip Select control
+ * @anchor spi_gpio_cs
```

Comment:
Not sure this is what you wanted? https://builds.zephyrproject.io/zephyr/pr/86900/docs/doxygen/html/group__spi__interface.html#ga27af5f053c5fd4e9eef0900406fbf854 (see weird Topics section)

Reply:
huh, I actually did build the docs locally and there was not this section, maybe I changed something since then. I dont know the doxygen that well to understand why did it make a topics section